### DISCUSSION TITLE
The getting started of myst.tools does not "get one started"

### DISCUSSION THREAD

**Discussion by jgalazm** [executablebooks/meta#910] [Q&A] [answered]
Hi there.

I am new to this project, and I am very excited to use it, however, when I tried to follow the guide at https://myst.tools,  after the "getting-started" I got stuck after the installation and did not know what to do. 

I see the guide continues to the "myst syntax overview", but I think that a "hello world" example with the main features and boilerplate would be helpful to connect the dots. My experience is that I keep reading about all these awesome features but I can not do them myself.

In particular, I would like to write a scientific article in markdown and build a pdf from it, get a Tex source that I can send to a journal, maybe build it as a website too, etc. It is not clear for me if I *should* use a `.yml` frontmatter or just the `.md` header, and the vscode extension did not make a difference when previewing the `.md` files. 

Do you know of simple examples that achieve these features? Do you think some of these could be put in the "getting started"?

Thanks a lot for the project though, its features sound really interesting.

**Comment by rowanc1** [comment]
Hi @jgalazm  -- thanks for the note, @choldgraf and I were meeting this morning to aim to improve the site documentation (it is still a relatively new project, and was previously part of the Curvenote CLI).

Probably the best docs we have for the tutorial walk through are still in Curvenote (see here for a [full tutorial](https://curvenote.com/docs/web/tutorial-deploy-local), replacing 'curvenote' with 'myst' should mostly get you there). That one uses a blog as the content. For a phd thesis [see here](https://phd.row1.ca/) ([source](https://github.com/rowanc1/phd-thesis)). For a paper [see here](https://www.stevejpurves.com/la-palma-earthquakes) ([source](https://github.com/stevejpurves/blog/tree/main/content/la-palma-earthquakes)).

Hope that helps -- please feel free to open issues directly here: https://github.com/executablebooks/mystjs/issues

Fixing issues on the onboarding process is top priority for us -- so we will respond fast!

**Comment by jgalazm** [comment]
Alright, I will try the tutorial and examples, thanks again!

**Reply by rowanc1** [reply]
Hi @jgalazm  -- I am running a webinar [next Wednesday](https://curvenote.com/blog/writing-a-scientific-paper-faster-myst-markdown) on getting started with mystjs for writing scientific papers, and using that as a deadline on myself to improve the docs. :) Would love to see you there! More on that [here](https://curvenote.com/blog/writing-a-scientific-paper-faster-myst-markdown).

**Reply by rowanc1** [reply, marked as answer]
Hi again @jgalazm -- we have put a bunch of effort over the last month into improving the first experience with myst-tools.org, especially the [documents quickstart](https://myst-tools.org/docs/mystjs/quickstart-myst-documents). Let us know if this is starting to meet the mark, would love any other feedback of where to improve!

https://myst-tools.org/docs/mystjs/quickstart-myst-documents

[![image](https://user-images.githubusercontent.com/913249/219072917-b12c1e59-0004-40a8-95f4-9bd3fa11d212.png)](https://myst-tools.org/docs/mystjs/quickstart-myst-documents)

**Reply by jgalazm** [reply]
Hi @rowanc1, thanks a lot, the guide is super clear and it definitely gives a good first experience.

Talking from my experience, as more of a latex user than a markdown user, what pulls me back from fully adopting myst in my workflow is the "synctex" feature that most latex editors have. From this getting started I get that, maybe, the experience is more of a frontend developer where you manually jump from code to live rendering of the app, focusing on one feature at a time. As a researcher, instead, I would like to be able to automatically/easily jump from a markdown line or equation to the rendered document and vice versa. This makes the writing much easier and productive than just guessing or re-reading the document in rendered or source code format. This is the "synctex" feature which you can find for example in the vscode  "Latex Workshop" extension. On my case cmd+click on the pdf preview puts the cursor in the corresponding line in the editor and pressing cmd+option+J from the editor scrolls the preview to the corresponding line and highlights it for a short time.

Do you guys have or plan on having such a feature in myst? I think that if you could add it to the getting started guides it would be a great addition for latex users like me.
